docs/page.js

```diff
@@ -140,6 +142,56 @@ function onDocumentLoad( event ) {
 
 	document.head.appendChild( prettify );
 
+	// Import module
+
+	var importSelect = document.querySelector( '.import-wrap select' );
+
+	if ( importSelect ) {
+
+		importSelect.addEventListener( 'change', function () {
+
+			var importBlocks = document.querySelectorAll( '.import-wrap [data-import]' );
+
+			for ( var i = 0; i < importBlocks.length; i ++ ) {
+
+				var block = importBlocks[ i ];
+
+				block.style.display = block.getAttribute( 'data-import' ) === importSelect.value ? '' : 'none';
+
+			}
+
+		} );
+
+	}
+
+	function formatImportSelect ( _, name, modulePath ) {
+
+		var globalPath = modulePath.replace( 'examples/jsm/', 'examples/js/' );
+
+		return [
+
+			'<h2>Import</h2>',
+			'<p>',
+			'	' + name + ' is optional, and must be included in your project before use. ',
+			'	For more information, see [link:#manual/introduction/Installation Installation - Examples].',
+			'</p>',
+			'',
```

Comment:
I think this will need to go `GLTFLoader.html` otherwise other languages won't be able to translate it.